modules/va_facilities/spec/requests/facilities_request_spec.rb

```diff
@@ -370,4 +370,68 @@ def query_params(url)
       expect(response).to have_http_status(:unprocessable_entity)
     end
   end
+
+  context 'mobile flag' do
+    it 'responds with a boolean mobile flag for VHA facilities' do
+      create :vha_648A4
+      get base_query_path + '/vha_648A4', params: nil, headers: accept_json
+      expect(response).to be_successful
+      expect(response.body).to be_a(String)
+      json = JSON.parse(response.body)
+      expect(json['data']['attributes']['mobile']).to eq(false)
+      expect(json['data']['attributes']['mobile']).to_not be_nil
```

Comment:
If the expectation on the preceding line passes, why would this line ever fail?

Reply:
This is a good point. I will reverse the expectations